lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua

```diff
@@ -90,3 +90,88 @@ e2function number array:indexOf(vector4 vec4)
     return indexOfPackedValue( this, vec4 )
 end
 
+e2function entity array:firstEntity()
+    return this[1]
+end
+e2function string array:firstString()
+    return this[1]
```

Comment:
```lua
return this[1] or ""
```
without this its possible for the function to return nil. e2 can't handle nil values.

Reply:
fixed

Reply:
Done